src/System.Drawing.Common/src/Unix/System.Drawing/Region.cs

```diff
@@ -642,6 +642,19 @@ internal IntPtr NativeObject
                 nativeRegion = value;
             }
         }
+
+        internal IntPtr _nativeRegion
```

Comment:
Are we not able to just rename the field to `_nativeRegion`?

Reply:
#23189 removed the Unix implementation of `Region`, so this problem fixed itself ;-)